src/hexo/themes/documentation/source/images/home-nellie.svg

```diff
@@ -0,0 +1 @@
{fill:#00A387;stroke:#2B2C2B;stroke-width:0.25;stroke-miterlimit:10;} .st35{fill:#864185;stroke:#2B2C2B;stroke-width:0.25;stroke-miterlimit:10;} .st36{fill:#BF4041;stroke:#2B2C2B;stroke-width:0.25;stroke-miterlimit:10;} .st37{fill:#CCCCCB;} .st38{fill:#4D4D4E;stroke:#4D4D4E;stroke-miterlimit:10;} .st39{fill:#E6E6E6;} .st40{fill:none;stroke:#000000;stroke-width:3;stroke-miterlimit:10;} .st41{stroke:#000000;stroke-miterlimit:10;} .st42{fill:#F2F2F3;stroke:#000000;stroke-width:3;stroke-miterlimit:10;} .st43{fill:none;} .st44{fill:#666766;stroke:#000000;stroke-miterlimit:10;} .st45{fill:none;stroke:#000000;stroke-miterlimit:10;} .st46{fill:#BF4041;} .st47{fill:#EF8422;} .st48{fill:#333333;} .st49{fill:#F8E8CF;stroke:#2B2C2B;stroke-width:0.25;stroke-miterlimit:10;} .st50{stroke:#000000;stroke-width:3;stroke-miterlimit:10;} .st51{fill:#4999D4;stroke:#2B2C2B;stroke-width:0.25;stroke-miterlimit:10;} .st52{clip-path:url(#SVGID_6_);} .st53{fill:#BB9166;stroke:#2B2C2B;stroke-width:0.25;stroke-miterlimit:10;} .st54{fill:#2B2C2B;stroke:#BB9166;stroke-width:0.25;stroke-miterlimit:10;}</style><path class="st0" d="M167.4 94.3C129.8 90.6 96 121 92 162.2c-.3 3.3-.4 6.6-.4 9.8-10.6 4.2-19.1 9.9-24.6 16.6-6.3-2.7-12.1-4.5-16.9-5.1l.1-.2 2-3.3.8-1.3c3.1-5.2.4-12.5-5.8-16.2l-7.9-4.7c-.6-.4-1.4-.3-1.7.3l-7.7 13.2c-2 3.4-1.3 7.8 1.4 11.2l-.1.2c-4.3-.7-8.5.8-10.5 4.2L13 200.1c-.3.5 0 1.3.6 1.7l7.9 4.7c6.3 3.7 13.9 2.5 17-2.7l1.4-2.4c4.1 6.4 11.6 14.3 20.7 21.2 2.4 1.8 4.4 3.1 7 4.9 8.1 5.3 16.1 9.5 23.4 12.2 12.7 5.2 28.5 8.3 45.7 8.3 16.5 0 31.7-2.9 44.2-7.7 5.2-1.9 10.1-4.4 14.6-7.5.9-.6 1.7-1.2 2.5-1.8.2-.2.4-.3.6-.5.5-.4 1-.7 1.4-1.1 15.4-12.3 26.3-31.4 28.4-53.6 3.9-41.5-23.3-77.9-61-81.5z"/><ellipse class="st1" cx="146.7" cy="132.3" rx="4.3" ry="3.7"/><circle class="st2" cx="147.4" cy="131.9" r=".7"/><ellipse class="st1" cx="194.2" cy="132.3" rx="4.3" ry="3.7"/><circle class="st2" cx="193.3" cy="131.9" r=".7"/><path class="st3" d="M171.2 153.3c-48.9 0-58.8 43.9-58.8 43.9-32.8 7.5-51.8 25.2-51.8 25.2 2.2 1.7 4.5 3.3 6.9 4.9 8.1 5.3 16.1 9.5 23.4 12.2 12.7 5.2 28.5 8.3 45.7 8.3 16.5 0 31.7-2.9 44.2-7.7 5.2-1.9 10.1-4.4 14.6-7.5.9-.6 1.7-1.2 2.5-1.8.2-.2.4-.3.6-.5.5-.4 1-.7 1.4-1.1 13.9-11.1 24.1-27.8 27.5-47.3-.3-1.7-8.4-28.6-56.2-28.6z"/><path class="st4" d="M165.5 145.3c3 2.2 7 2.2 10 .1l.2-.1"/><path class="st25" d="M119.8 180.9s17.1 9.7 25.4-2.3c.7-1 .4-2.3-.7-2.8l-19.5-10"/><path class="st3" d="M191.7 14.7L178 112.1l-.7 5.2c-2.9 2.3-7 1.9-9.9.3l-.5-.3 24.8-102.6z"/><path class="st15" d="M166.9 117.4s9.1-.7 11.2-5.7M169.5 108.9c2.6-.4 8.2-1.8 9.9-5.5M171 100.6c2.8-.5 7.9-1.9 9.4-5.5M173.4 91.8c2.8-.7 6.9-2.1 8.2-5.2M175.4 83.2c2.7-.8 6-2.2 7.2-5M177.2 74.9c2.7-.8 5.7-2.2 6.8-4.9M179.2 66.3c2.4-.8 4.8-2.2 5.8-4.5M181.3 57.6c2.1-.9 4.1-2.2 4.9-4.2M183.7 48.6c1.5-.8 3.2-2.5 3.9-4.1M188.7 36.3c-.5 1.2-1.9 2.6-2.9 3.4"/><path class="st0" d="M118.7 180.5s17.1 9.7 25.4-2.3c.7-1 .4-2.3-.7-2.8L123 164.5M228.2 176s18-7.9 13.4-21.6c-.4-1.1-1.6-1.7-2.7-1.1L218 163.4"/></svg>
```

Comment:
Running it through https://jakearchibald.github.io/svgomg/ can get some savings (1.4k - 0.55% saving), and I'm sure @alrra can do a bit more:
```
<svg id="Layer_1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 262.2 270.1"><style>.st0{fill:#929ACD;stroke:#2B2C2B;stroke-width:0.25;stroke-miterlimit:10;} .st1{fill:#231F20;} .st2{fill:#FFFFFF;} .st3{fill:#D7DAEE;stroke:#2B2C2B;stroke-width:0.25;stroke-miterlimit:10;} .st4{fill:none;stroke:#1A1A1A;stroke-width:0.75;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:10;} .st5{fill:#262626;} .st6{fill:#CFCFCE;stroke:#000000;stroke-miterlimit:10;} .st7{fill:#20242B;} .st8{fill:#7C5E2A;} .st9{clip-path:url(#SVGID_2_);fill:#929ACD;stroke:#2B2C2B;stroke-width:0.25;stroke-miterlimit:10;} .st10{clip-path:url(#SVGID_2_);fill:#231F20;} .st11{clip-path:url(#SVGID_2_);fill:#FFFFFF;} .st12{clip-path:url(#SVGID_2_);fill:#D7DAEE;stroke:#2B2C2B;stroke-width:0.25;stroke-miterlimit:10;} .st13{clip-path:url(#SVGID_2_);fill:none;stroke:#1A1A1A;stroke-width:0.75;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:10;} .st14{clip-path:url(#SVGID_2_);} .st15{fill:none;stroke:#6B6C6C;stroke-width:0.25;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:10;} .st16{clip-path:url(#SVGID_2_);fill:none;} .st17{clip-path:url(#SVGID_2_);fill:none;stroke:#4F5681;stroke-width:0.25;stroke-miterlimit:10;} .st18{clip-path:url(#SVGID_2_);fill:#E6ECF1;stroke:#4F5681;stroke-width:0.25;stroke-miterlimit:10;} .st19{fill:#DE625C;stroke:#2B2C2B;stroke-width:0.25;stroke-miterlimit:10;} .st20{clip-path:url(#SVGID_4_);} .st21{fill:#3F458C;} .st22{fill:#D7D6D6;stroke:#56504F;stroke-width:0.5;stroke-miterlimit:10;} .st23{fill:#F26F6A;} .st24{fill:#C6665E;stroke:#2B2C2B;stroke-width:0.25;stroke-miterlimit:10;} .st25{fill:#3C5CAA;} .st26{fill:#FFFFFF;stroke:#2B2C2B;stroke-width:0.25;stroke-miterlimit:10;} .st27{fill:#908FB3;} .st28{fill:#4F5681;} .st29{fill:#F7F7F7;stroke:#000000;stroke-width:0.25;stroke-miterlimit:10;} .st30{fill:#ABADB0;} .st31{fill:#636466;} .st32{fill:#929ACD;stroke:#2B2C2B;stroke-width:0.25;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:10;} .st33{fill:none;stroke:#2B2C2B;stroke-width:0.25;stroke-miterlimit:10;} .st34{fill:#00A387;stroke:#2B2C2B;stroke-width:0.25;stroke-miterlimit:10;} .st35{fill:#864185;stroke:#2B2C2B;stroke-width:0.25;stroke-miterlimit:10;} .st36{fill:#BF4041;stroke:#2B2C2B;stroke-width:0.25;stroke-miterlimit:10;} .st37{fill:#CCCCCB;} .st38{fill:#4D4D4E;stroke:#4D4D4E;stroke-miterlimit:10;} .st39{fill:#E6E6E6;} .st40{fill:none;stroke:#000000;stroke-width:3;stroke-miterlimit:10;} .st41{stroke:#000000;stroke-miterlimit:10;} .st42{fill:#F2F2F3;stroke:#000000;stroke-width:3;stroke-miterlimit:10;} .st43{fill:none;} .st44{fill:#666766;stroke:#000000;stroke-miterlimit:10;} .st45{fill:none;stroke:#000000;stroke-miterlimit:10;} .st46{fill:#BF4041;} .st47{fill:#EF8422;} .st48{fill:#333333;} .st49{fill:#F8E8CF;stroke:#2B2C2B;stroke-width:0.25;stroke-miterlimit:10;} .st50{stroke:#000000;stroke-width:3;stroke-miterlimit:10;} .st51{fill:#4999D4;stroke:#2B2C2B;stroke-width:0.25;stroke-miterlimit:10;} .st52{clip-path:url(#SVGID_6_);} .st53{fill:#BB9166;stroke:#2B2C2B;stroke-width:0.25;stroke-miterlimit:10;} .st54{fill:#2B2C2B;stroke:#BB9166;stroke-width:0.25;stroke-miterlimit:10;}</style><path class="st0" d="M167.4 94.3C129.8 90.6 96 121 92 162.2c-.3 3.3-.4 6.6-.4 9.8-10.6 4.2-19.1 9.9-24.6 16.6-6.3-2.7-12.1-4.5-16.9-5.1l.1-.2 2-3.3.8-1.3c3.1-5.2.4-12.5-5.8-16.2l-7.9-4.7c-.6-.4-1.4-.3-1.7.3l-7.7 13.2c-2 3.4-1.3 7.8 1.4 11.2l-.1.2c-4.3-.7-8.5.8-10.5 4.2L13 200.1c-.3.5 0 1.3.6 1.7l7.9 4.7c6.3 3.7 13.9 2.5 17-2.7l1.4-2.4c4.1 6.4 11.6 14.3 20.7 21.2 2.4 1.8 4.4 3.1 7 4.9 8.1 5.3 16.1 9.5 23.4 12.2 12.7 5.2 28.5 8.3 45.7 8.3 16.5 0 31.7-2.9 44.2-7.7 5.2-1.9 10.1-4.4 14.6-7.5.9-.6 1.7-1.2 2.5-1.8.2-.2.4-.3.6-.5.5-.4 1-.7 1.4-1.1 15.4-12.3 26.3-31.4 28.4-53.6 3.9-41.5-23.3-77.9-61-81.5z"/><ellipse class="st1" cx="146.7" cy="132.3" rx="4.3" ry="3.7"/><circle class="st2" cx="147.4" cy="131.9" r=".7"/><ellipse class="st1" cx="194.2" cy="132.3" rx="4.3" ry="3.7"/><circle class="st2" cx="193.3" cy="131.9" r=".7"/><path class="st3" d="M171.2 153.3c-48.9 0-58.8 43.9-58.8 43.9-32.8 7.5-51.8 25.2-51.8 25.2 2.2 1.7 4.5 3.3 6.9 4.9 8.1 5.3 16.1 9.5 23.4 12.2 12.7 5.2 28.5 8.3 45.7 8.3 16.5 0 31.7-2.9 44.2-7.7 5.2-1.9 10.1-4.4 14.6-7.5.9-.6 1.7-1.2 2.5-1.8.2-.2.4-.3.6-.5.5-.4 1-.7 1.4-1.1 13.9-11.1 24.1-27.8 27.5-47.3-.3-1.7-8.4-28.6-56.2-28.6z"/><path class="st4" d="M165.5 145.3c3 2.2 7 2.2 10 .1l.2-.1"/><path class="st25" d="M119.8 180.9s17.1 9.7 25.4-2.3c.7-1 .4-2.3-.7-2.8l-19.5-10"/><path class="st3" d="M191.7 14.7L178 112.1l-.7 5.2c-2.9 2.3-7 1.9-9.9.3l-.5-.3 24.8-102.6z"/><path class="st15" d="M166.9 117.4s9.1-.7 11.2-5.7m-8.6-2.8c2.6-.4 8.2-1.8 9.9-5.5m-8.4-2.8c2.8-.5 7.9-1.9 9.4-5.5m-7-3.3c2.8-.7 6.9-2.1 8.2-5.2m-6.2-3.4c2.7-.8 6-2.2 7.2-5m-5.4-3.3c2.7-.8 5.7-2.2 6.8-4.9m-4.8-3.7c2.4-.8 4.8-2.2 5.8-4.5m-3.7-4.2c2.1-.9 4.1-2.2 4.9-4.2m-2.5-4.8c1.5-.8 3.2-2.5 3.9-4.1m1.1-8.2c-.5 1.2-1.9 2.6-2.9 3.4"/><path class="st0" d="M118.7 180.5s17.1 9.7 25.4-2.3c.7-1 .4-2.3-.7-2.8L123 164.5M228.2 176s18-7.9 13.4-21.6c-.4-1.1-1.6-1.7-2.7-1.1L218 163.4"/></svg>
```